checker/check-request-property-type-changed.go

```diff
@@ -52,11 +59,16 @@ func RequestPropertyTypeChangedCheck(diffReport *diff.Diff, operationsSources *d
 						schemaDiff := propertyDiff
 						typeDiff := schemaDiff.TypeDiff
 						formatDiff := schemaDiff.FormatDiff
+						level := INFO
+
 						if breakingTypeFormatChangedInRequestProperty(typeDiff, formatDiff, mediaType, schemaDiff) {
+							level = ERR
+						}
```

Comment:
I reccommend to replace all the places where level is a function of a boolean condition to a function that can be called directly from the ApiChange constructor, for example:

```
ApiChange{
    Id:          "request-property-type-changed",
    Level:       conditionalLevel(breakingTypeFormatChangedInRequestProperty(typeDiff, formatDiff, mediaType, schemaDiff)),
    Text:        fmt.Sprintf(config.i18n("request-property-type-changed"), ColorizedValue(propertyFullName(propertyPath, propertyName)), empty2none(typeDiff.From), empty2none(formatDiff.From), empty2none(typeDiff.To), empty2none(formatDiff.To)),
    Operation:   operation,
    OperationId: operationItem.Revision.OperationID,
    Path:        path,
    Source:      source,
}
```

Reply:
done, thanks